asab/api/web_handler.py

```diff
@@ -4,47 +4,62 @@
 
 
 class APIWebHandler(object):
+    def __init__(self, app, webapp, log_handler):
+        self.App = app
 
-	def __init__(self, app, webapp, log_handler):
-		# Add routes
-		webapp.router.add_get('/asab/v1/environ', self.environ)
-		webapp.router.add_get('/asab/v1/config', self.config)
```

Comment:
Oh, we use TABs as indentation ... there is actually Flake8 profile that should help you with the code style.